CHANGELOG.md

```diff
@@ -866,7 +866,7 @@
 * ARROW-5809 - [Rust] Dockerize (add to docker-compose) Rust Travis CI build
 * ARROW-5831 - [Release] Migrate and improve binary release verification script
 * ARROW-5855 - [Python] Add support for Duration type
-* ARROW-5859 - [Python] Support ExtentionType on conversion to numpy/pandas
+* ARROW-5859 - [Python] Support ExtensionType on conversion to numpy/pandas
```

Comment:
According to https://github.com/apache/arrow/pull/7274#discussion_r430541033, it looks necessary to revert all of changes in `CHANGELOG.md`.

Reply:
OK, Thanks, now all the modification in `CHANGELOD.md` have been reverted.